### ISSUE TITLE
NuGet fails to install with VS C# 2010 Express

### ISSUE THREAD

**Issue by Cosmogeek** [zeromq/clrzmq#63]
I am trying to get Clrzmq.dll. The only way I can find to do this is via NuGet. However, I am not able to install NuGet. I think it will not install on the Express versions of Visual Studio. Is there a way I can just download Clrzmq.dll and manually install it?

Has anybody been able to use ZeroMQ with VS C# 2010 Express? If so, how did you get Clrzmq installed?

Thanks for any help.

**Comment by Cosmogeek** [comment]
I got past this by installing Visual Web Developer 2010, Installed NuGet into this. Now I can load my project into VWD to add the clrzmq package, and then go back to VS C# 2010 Express. However, I still can't compile because ZMQContext is not recognized. I'm just trying to compile the Hello World example. Does anybody know what I might be missing? The compiler chokes on this line:
            using (ZmqContext context = ZmqContext.Create())
but is now getting past :
using ZMQ:

BTW, as best I can figure out, you can't use NuGet package manager with VS C# 2010 Express.

**Comment by Cosmogeek** [comment]
OK I figured it out. The ZMQ syntax has changed since those sample programs were published.

For example:

using (var context = new ZMQ.Context(1))

is the proper way to set up a context. I am still looking for the correct documentation, but otherwise I have finally got this working. I feel like I had more problems than I should have. For something as simple as ZeroMQ, the installation was difficult and requires a cumbersom workaround that is not documented. Also, the changes in the API don't seem wel documented. But as far as providing a solution to my problem I am working on, I think this is just the right technology.
